lua/neogit/process.lua

```diff
@@ -36,6 +36,8 @@ local function spawn(options, cb)
     table.insert(params.env, string.format("%s=%s", "GNUPGHOME", os.getenv("GNUPGHOME") or ""))
     table.insert(params.env, string.format("%s=%s", "NVIM", vim.v.servername))
     table.insert(params.env, string.format("%s=%s", "PATH", os.getenv("PATH")))
+    table.insert(params.env, string.format("%s=%s", "SSH_AUTH_SOCK", os.getenv("SSH_AUTH_SOCK" or "")))
```

Comment:
shouldn't this be `os.getenv("SSH_AUTH_SOCK") or ""`? same on the other line

Reply:
Yes that is a mistake on my part, I'll amend the commit with this fixed.